api.js

```diff
@@ -258,6 +259,11 @@ class Api {
     return this
   }
 
+  unexpectedErrorFormatter (fn) {
+    this._errorFormatter = fn
```

Comment:
I kinda wish the name of the variable more closely resembled the name of the Api method, but I also prefer shorter names to longer ones.

I don't know, this is probably fine. (No action needed.)